src/world/Explosion.php

```diff
@@ -54,6 +61,9 @@ class Explosion{
 	/** @var Block[] */
 	public array $affectedBlocks = [];
 	public float $stepLen = 0.3;
+	private float $fireChance = 0.0;
```

Comment:
I would also like to have an ability to pass fire chance in the explosion constructor